appsec-kit-demo-v8/osv-service-v1-swagger.json

```diff
@@ -0,0 +1,630 @@
+{
```

Comment:
I'm going to remove this swagger specification file as it has only been added to represent the available OSV API endpoints.